src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedPostConnection.java

```diff
@@ -82,4 +82,13 @@ public Connection<Post> retrievePostsByAuthor(URN authorURN, Integer count) {
     return linkedinClient.fetchConnection(POSTS, Post.class,
         parameters.toArray(new Parameter[parameters.size()]));
   }
+  
+  /**
+   * Create a post.
+   * @see <a href="https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/posts-api#create-a-post">Create a Post</a>
+   * @param post The UGC Post to share
+   */
+  public void createPost(Post post) {
```

Comment:
Can we return the Post created? We need to get at least the post URN in main

Reply:
At the moment, LinkedIn doesn't return a response when creating a post with the new versioned API. I've opened a ticket with them requesting that they do return a response as it is in the older API version

Reply:
It seems the post id can be found in response headers
https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/posts-api?view=li-lms-2022-12&tabs=http#post-creation-response

Please have a look if it can be retrieved. May need to add new functions to the linkedin client

Reply:
Nice catch. I'll retrieve the post URN from the headers in the meantime. We can always iterate once (if) LinkedIn makes the change. I assume their turnaround time is not that quick :woozy_face: